chapters/ch16.md

```diff
@@ -1,70 +1,71 @@
 
 * Errors due to mislabeled examples....... 0.6% (6% of dev set errors.)
 ->
-
+* Những lỗi gây ra bởi các mẫu bị gán nhãn nhầm....... 0.6% (6% các lỗi trong tập phát triển.)
 
 * Errors due to other causes..................... 9.4% (94% of dev set errors)
 ->
-
+* Những lỗi do những nguyên nhân khác..................... 9.4% (94% các lỗi trong tập phát triển)
 
 Here, the 0.6% inaccuracy due to mislabeling might not be significant enough relative to the 9.4% of errors you could be improving. There is no harm in manually fixing the mislabeled images in the dev set, but it is not crucial to do so: It might be fine not knowing whether your system has 10% or 9.4% overall error.
 ->
-
+Ở đây, tỷ lệ 0.6% sai do gán nhầm nhãn có thể không quan trọng bằng với tỷ lệ 9.4% các lỗi mà bạn có thể cải thiện. Không có một tác hại nào trong việc sửa thủ công những ảnh bị gán nhẫn nhầm trong tập phát triển cả, nhưng nó không quá quan trọng để làm vậy: Có thể là ổn khi không biết liệu hệ thống của bạn có 10% hay 9.4% lỗi chung:
 
 Suppose you keep improving the cat classifier and reach the following performance:
 ->
-
+Giả sử bạn tiếp tục cải tiến bộ phân loại mèo và đạt được những hiệu suất sau:
 
 * Overall accuracy on dev set.................... 98.0% (2.0% overall error.)
 ->
-
+* Độ chính xác chung trên tập phát triển.................... 98.0% (2.0% lỗi chung.)
 
 * Errors due to mislabeled examples....... 0.6%. (30% of dev set errors.)
 ->
-
+* Những lỗi gây ra do các mẫu bị gán nhãn nhầm....... 0.6%. (30% các lỗi trong tập phát triển.)
 
 * Errors due to other causes..................... 1.4% (70% of dev set errors)
 ->
-
+* Những lỗi do những nguyên nhân khác..................... 1.4% (70% các lỗi trong tập phát triển)
 
 30% of your errors are due to the mislabeled dev set images, adding significant error to your estimates of accuracy. It is now worthwhile to improve the quality of the labels in the dev set. Tackling the mislabeled examples will help you figure out if a classifier’s error is closer to 1.4% or 2%—a significant relative difference.
 ->
-
+30% những lỗi của bạn là do những ảnh bị gán nhãn nhầm trong tập phát triển gây ra, thêm vào một lượng lớn lỗi cho các đánh giá độ chính xác của bạn. Bây giờ, nó rất đáng giá khi cải thiện chất lượng của các nhãn trong tập phát triển. Tấn công vào những mẫu bị gán nhãn nhầm sẽ giúp bạn hình dung được nếu lỗi của một bộ phân loại gần hơn với 1.4% hay 2%—một sự khác biệt đáng kể.
 
 It is not uncommon to start off tolerating some mislabeled dev/test set examples, only later to change your mind as your system improves so that the fraction of mislabeled examples grows relative to the total set of errors.
 ->
-
+Nó không bất bình thường khi bắt đầu bạn bỏ qua một vài mẫu bị gán nhãn nhầm của tập phát triển/tập kiểm tra, chỉ sau này bạn nên thay đổi suy nghĩ của bạn khi hệ thống của bạn phát triển dẫn tới tỷ lệ những mẫu bị gán nhãn nhầm tăng theo toàn bộ tập các lỗi.
```

Comment:
- ~~Nó không bất bình thường khi bắt đầu bạn bỏ qua một vài mẫu bị gán nhãn nhầm của tập phát triển/tập kiểm tra~~

Nói chung muốn đọc cho thuận phải chịu khó đổi lại cấu trúc câu một chút. VD: 
Việc chấp nhận một vài mẫu bị gán nhãn sai (...) khi bắt đầu không có gì lạ
Không có gì lạ khi bắt đầu với việc chấp nhận một vài mẫu bị gán nhãn sai

- "only later to change your mind" --> để rồi sau này đổi ý
- "tăng tỉ lệ theo"